tsconfig.json

```diff
@@ -10,6 +10,14 @@
 		"resolveJsonModule": true,
 		"target": "esnext",
 
+		/* Project Imports */
+		"baseUrl": ".",
+		"paths": {
+			"@wordpress/*": [
+				"packages/*/src/index.js"
```

Comment:
Not all packages follow this pattern, namely `is-shallow-equal`:

https://github.com/WordPress/gutenberg/blob/d6b36016340f1ad769df5e972f04ec19e08c3e92/packages/is-shallow-equal/index.js#L24-L36

An established, consistent convention for packages to follow seems beneficial.

Reply:
Those which doesn't need to be transpiled don't have `src` folder.

Reply:
> Those which don't need to be transpiled don't have `src` folder.

Yes, this is the intended convention. Any package should include its main source at `index.js` if not transpiled, otherwise at `src/index.js`.